pkg/receive/handler.go

```diff
@@ -339,6 +340,51 @@ func (h *Handler) receiveHTTP(w http.ResponseWriter, r *http.Request) {
 		tenant = h.options.DefaultTenantID
 	}
 
+	// Checking certs for possible value.
+	if h.options.TenantAttribute != "" {
+		if len(r.TLS.PeerCertificates) > 0 {
+			// First cert is the leaf authenticated against.
+			cert := r.TLS.PeerCertificates[0]
+
+			switch h.options.TenantAttribute {
+
+			case "organization":
```

Comment:
Let's turn these into an enum to make them:
1. easier to discover the options for code using the package and
2. harder to mess-up with a typo.